tlsn/benches/bin/plot.rs

```diff
@@ -13,25 +14,44 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let mut rdr = csv::Reader::from_path(csv_file)?;
 
     // Prepare data for plotting
-    let mut data = Vec::new();
-    for result in rdr.deserialize() {
-        let record: Metrics = result?;
-        // Summing upload_delay and download_delay for the x-axis value
-        let total_delay = record.upload_delay + record.download_delay;
-        // Using runtime for the y-axis value
-        if record.name == "latency" {
-            data.push(vec![total_delay as f32, record.runtime as f32]);
-        }
-    }
+    let data: Vec<Vec<f32>> = rdr
+        .deserialize::<Metrics>()
+        .filter_map(Result::ok)
```

Comment:
nice stuff, but we probably want to panic and exit if there is any data that can't be deserialized? Else the plot might be inaccurate?